scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -108,7 +108,7 @@ RUN set -ex; \
 	cd hera; \
 	mkdir build; \
 	cd build; \
-	cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX="/usr" ..; \
+	cmake -G Ninja -DHERA_DEBUGGING=OFF -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX="/usr" ..; \
```

Comment:
Debugging is off by default, so no need to specify this?

Reply:
I think `HERA_DEBUGGING` is by default on. See https://github.com/ewasm/hera/blob/65efd8c9b6ddaef518d55d42e49c6c4329a3ad0c/src/CMakeLists.txt#L31